Resolved empty URL redirects, file_size handling and file details redirection

lakshya-20 opened the pull request:
# Description

- Now, empty URLs will not redirect to download page.
- Resolved file handling as Andoird client is storing file size in bytes so modified web client accordingly.
- Enhanced the file listing, now the user can go to the file details component by clicking the file card.

Fixes #51  #55  #58 

## Type of change

Please delete options that are not relevant.

- [X] Updated UI/UX
- [X] Improved the business logic of code
- [ ] Added new feature
- [ ] Other

## Checklist:

- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings

lakshya-20 committed:
Resolved empty URL redirects, file_size handling and file details redirection

danishjamal104 reviewed (commented):
@lakshya-20, have you removed that circle button that was allowing the user to see file details? or you are planning it to be there.

lakshya-20 commented:
> @lakshya-20, have you removed that circle button that was allowing the user to see file details? or you are planning it to be there.

I have removed that button and added the navigation to the card.
